spec/factories/users.rb

```diff
@@ -4,5 +4,8 @@
   factory :user do
     email { Faker::Internet.email }
     password { Faker::Internet.password(min_length: 6) }
+    name { Faker::Name.first_name }
+    birth { Faker::Date.birthday }
```

Comment:
make sure that the generated birthday is at least 18 years prior to the current date. Otherwise, this factory can generate invalid users.